contracts/src/v0.8/ccip/test/feeQuoter/FeeQuoter/FeeQuoter.applyDestChainConfigUpdates.t.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.24;
 
-import {FeeQuoter} from "../../FeeQuoter.sol";
```

Comment:
FeeQuoter is a top level contract so you don't need a feeQuoter/FeeQuoter folder, just a FeeQuoter one

Reply:
https://github.com/smartcontractkit/chainlink/pull/15165/commits/b039d1fbb0866525e68fc31785d48bdb06ef05c0